tests/lib/lockfree/src/test_mpsc.c

```diff
@@ -4,32 +4,31 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "zephyr/irq.h"
 #include <zephyr/ztest.h>
 #include <zephyr/kernel.h>
 #include <zephyr/sys/util_loops.h>
 #include <zephyr/timing/timing.h>
-#include <zephyr/rtio/rtio_spsc.h>
-#include <zephyr/rtio/rtio_mpsc.h>
+#include <zephyr/sys/spsc_lockfree.h>
+#include <zephyr/sys/mpsc_lockfree.h>
```

Comment:
The fact that they are both called `_lockfree.h` helps grouping them semantically, and suggest the kind of feature they provide.